test/index.js

```diff
@@ -35,35 +40,33 @@ test('podcast', function(t) {
 		itunesSummary: 'All About Everything is a show about everything. Each week we dive into any subject known to man and talk about it as much as we can. Look for our podcast in the Podcasts app or in the iTunes Store',
 		itunesOwner: {
 			name: 'John Doe',
-			email: 'john.doe@example.com'
+			email: 'john.doe@example.com',
 		},
 		itunesImage: 'http://example.com/podcasts/everything/AllAboutEverything.jpg',
 		itunesCategory: [{
 			text: 'Technology',
 			subcats: [{
 				text: 'Software',
 				subcats: [{
-					text: 'node.js'
-				}]
-			}]
+					text: 'node.js',
+				}],
+			}],
 		}],
```

Comment:
Why the trailing commas?

Reply:
Mostly all [this stuff](https://dnlserrano.github.io/2015/07/29/a-case-for-trailing-commas/) - personally, I'm to the point where I think I shouldn't have to type the commas at all, a newline should be enough inside of object literals, just like semiclons!

I added a rule to my eslint autoformatter for [dangling commas](https://eslint.org/docs/rules/comma-dangle).

Reply:
I don't mind them, I was just making sure it was intentional.

The argument that they make git diffs cleaner doesn't really apply in this case :stuck_out_tongue:

Reply:
lol, true